octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py

```diff
@@ -246,14 +260,42 @@ def get_exchange_pair(self, pair) -> str:
     """
     Private methods
     """
+
     def _create_client(self):
-        self.client = cryptofeed.FeedHandler()
+        """
+        Creates cryptofeed client instance
+        """
+        self._create_client_config()
+        self.client = cryptofeed.FeedHandler(config=self.client_config)
+
+    def _create_client_config(self):
+        """
+        Creates cryptofeed config
+        """
+        self.client_config = cryptofeed_config.Config()
+        self.client_config.config.get('log', {})['filename'] = ""
+        self.client_config.config.get('rest', {}).get('log', {})['filename'] = ""
```

Comment:
why do we need this ?

Reply:
Cryptofeed logs are currently written in `feedhandler.log`. I disabled logs saving by using an empty file name